start.sh

```diff
@@ -1,12 +1,18 @@
-set -e 
+set -e
 
 if [ ${RAILS_ENV:-development} != "production" ]; then
   bundle check || bundle
+  yarn
 fi
 
-echo "starting rails"
-rm -f tmp/pids/server.pid
-bundle exec rails db:create
-bundle exec rails db:migrate
-bundle exec rails solr:init
-bundle exec rails s -b '0.0.0.0'
+if [ ${APP_ROLE:-app} == "sidekiq" ]; then
+    echo "starting sidekiq"
+    /app/bin/sidekiq
```

Comment:
@whereismyjetpack I'm assuming this means that if there's a docker container (?) running with the APP_ROLE of sidekiq, then it just starts sidekiq?

Reply:
This is correct. the deployment for sidekiq sets that variable, and then runs as that "role" this allows both "roles" to share the same entrypoint to bootstrap the environment